extensions/wikia/Piggyback/templates/Piggyback_form.php

```diff
@@ -0,0 +1,59 @@
+<?php if( !empty( $message ) ): ?>
+<div class="<?= $messagetype ?>box">
+	<?php if ( $messagetype == 'error' ):  ?>
+		<h2><?= wfMessage( 'loginerror' )->escaped() ?></h2>
+	<?php endif ?>
+	<?= $message ?>
+</div>
+<div class="visualClear"></div>
+<?php endif ?>
+
+<div id="userloginForm">
+	<form name="userlogin" method="post" action="<?= $actionlogin ?>">
+		<h2><?= wfMessage( 'login' )->escaped() ?></h2>
+
+		<table>
+			<tr>
+				<td class="mw-label"><label
+						for='wpName1'><?= wfMessage( 'piggyback-otherusername' )->escaped() ?></label></td>
+				<td class="mw-input">
+					<input type='text' class='loginText' name="wpOtherName" id="wpOtherName1"
+						   value="<?= $otherName ?>" size='20'/>
+				</td>
+			</tr>
+		</table>
+		<div id="piggyback_userloginprompt"><p><?= wfMessage( 'piggyback-loginprompt' )->escaped() ?></p></div>
+
+		<div id="userloginprompt"><?= wfMessage( 'loginprompt' )->escaped() ?></div>
+		<table>
+			<tr>
+				<td class="mw-label"><label for='wpName1'><?= wfMessage( 'yourname' )->escaped() ?></label></td>
+				<td class="mw-input">
+					<input type='text' class='loginText' name="wpName" id="wpName1" value="<?= $name ?>" size='20'/>
```

Comment:
+1 for ```$name``` and ```$uselang```.
I don't think we need to escape tokens, while they are created by our application.